[Fixed Mobile table issue overlap]

hitesh-wagento opened the pull request:
## Purpose of this PR
Resolved Tables overflow in Mobile view

<!-- REQUIRED Describe the goal and the type of changes this PR covers. -->

See https://magento-research.github.io/pwa-studio/technologies/tools-libraries/  Tables overflow in Mobile view



## Affected URLs
https://magento-research.github.io/pwa-studio/technologies/tools-libraries/
<!-- REQUIRED List the URLs you are changing. Not needed for large numbers of files. -->

## Actual Result
![actual](https://user-images.githubusercontent.com/23443991/57771525-909f3800-7730-11e9-9fbc-ce85411fa5f3.png)

## Expected Result
![expected](https://user-images.githubusercontent.com/23443991/57771527-909f3800-7730-11e9-888d-dd3866838067.png)

_____________________________________________________

I have below changes in `/devdocs-theme/_scss/partials/_table.scss` file
`@media ( $small-screen) {
  th {
    word-wrap: break-word;
  }
}`

But in pull request it was show me only one file changes : `assets/css/app.css`
Can you please review and let me know if I am missing any steps.

hitesh-wagento commented:
@belbiy 

Can you please review this PR is correct?

Thanks

belbiy commented:
There is 2 issues with this PR:
1. You're changing the minified CSS file without modifying the source file. Your code will be overwritten by any other change since the minified file is generated. The source files for CSS are located in `_scss` folder and you'll want to use `npm build:css` command to minify them.

2. The issue you're trying to fix is specific to content in PWA repository and should be fixed there. You can add CSS classnames in that project as well.

hitesh-wagento commented:
Hi @belbiy 

I added below in /devdocs-theme/_scss/partials/_table.scss file because I need to word break in mobile view to resolved scroll in mobile.
`@media ( $small-screen) { th { word-wrap: break-word; } }`

And I have also added a class in https://github.com/magento-research/pwa-studio/pull/1263/commits/c56d1f5b17bbabe431ec019ad72e1c58f1f787f8.

You can see these changes in https://github.com/magento-research/pwa-studio/pull/1263

Magento docs all css coming from **devdocs-theme** so I have changed in it.

Can you please suggest how can I resolve these different repo changes in one PR ?

Thanks
